scripts/shell/new_genesis.sh

```
@@ -0,0 +1,121 @@
+    echo "Adding validator: $key ($ACCOUNT)"
+    printf "$KEYRING_SECRET\n" | treasurenetd add-genesis-account \
+        --trace \
+        --keyring-backend file \
+        "$ACCOUNT" \
+        200000000000000000000aunit
+done
+
+
+# Phase 3: Finalize genesis file
+echo "Creating final genesis file..."
+cd /data/genesis-validator-1/.treasurenetd/config/gentx
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)